lesson_19_hm_web/app.py

```diff
@@ -0,0 +1,26 @@
+from sqlalchemy import create_engine, Table, MetaData, Column, Integer, String
+from sqlalchemy.orm import DeclarativeBase
+from sqlalchemy.orm import Session
+from flask import Flask
+
+metadata = MetaData()
+engine = create_engine('postgresql+psycopg2://Somwl:1548@localhost/lesson_19')
+session = Session(bind=engine)
+
+
+class Base(DeclarativeBase):
+    pass
+
+
+class Intake(Base):
+    __tablename__ = "person_intake"
+    id = Column("id", Integer(),  primary_key=True)
+    user_name = Column("user", String(255), nullable=False, unique=True)
```

Comment:
it doesnt number.. doesnt match the contract

Reply:
what exactly is not a number

Reply:
User_name - should be a number if we look your contract..